pkg/env/dev.go

```diff
@@ -297,3 +297,7 @@ func (d *dev) CreateARMResourceGroupRoleAssignment(ctx context.Context, fpAuthor
 		return true, nil
 	})
 }
+
+func (d *dev) E2EStorageAccountName() string {
+	return "arov4e2eint"
```

Comment:
This is already created and always be the one we will use

Reply:
@julienstroheker Will we use the same account for INT and Prod ?

Reply:
no this will be different for int and prod
But we will have 1 unique storage account for all the prod region